tests/test_fs_drift.sh

```diff
@@ -33,21 +31,17 @@ function functional_test_fs_drift {
       count=$((count + 1))
     fi
   done
-
   echo fsdrift_pod $fs_drift_pod
-  wait_for "kubectl wait --for=condition=Initialized pods/$fsdrift_pod \
-    --namespace my-ripsaw --timeout=200s" "200s" $fsdrift_pod
-  wait_for "kubectl wait --for=condition=complete -l app=fs-drift-benchmark-$uuid jobs \
-    --namespace my-ripsaw --timeout=100s" "200s" $fsdrift_pod
-  sleep 5
-  # ensuring the run has actually happened
-  kubectl logs "$fsdrift_pod" --namespace my-ripsaw | grep "RUN STATUS"
-  if [ $? = 0 ] ; then
-    echo "fs-drift test: Success" 
-  else 
-    echo fs-drift test FAILURE 
-    exit 1 
-  fi
+  wait_for "kubectl wait --for=condition=Initialized pods/$fsdrift_pod -n my-ripsaw --timeout=200s" "200s" $fsdrift_pod
+  wait_for "kubectl wait --for=condition=complete -l app=fs-drift-benchmark-$uuid jobs -n my-ripsaw --timeout=100s" "200s" $fsdrift_pod
+  # Print logs and check status
+  kubectl logs "$fsdrift_pod" -n my-ripsaw
+  kubectl logs "$fsdrift_pod" -n my-ripsaw | grep "RUN STATUS"
```

Comment:
do we need to check status of this grep command and fail the test if grep does not find this?  Like:
```
kubectl logs "$fsdrift_pod" -n my-ripsaw | grep "RUN STATUS" || exit $NOTOK
```

Reply:
grep already returns 1 if there isn't any match, in addition all test scripts have a trap to capture ERROR signals I think it wouldn't be necessary.

Reply:
yeah I see that now, the set -xe... on line 1,  this comment is resolved.